bees/discordbee/discordbee.go

```diff
@@ -0,0 +1,143 @@
+/*
+ *    Copyright (C) 2019      CalmBit
+ *                  2014-2019 Christian Muehlhaeuser
+ *
+ *    This program is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU Affero General Public License as published
+ *    by the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    This program is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *    Authors:
+ *      CalmBit <calmbit@posto.net>
+ *      Christian Muehlhaeuser <muesli@gmail.com>
+ */
+
+// Package discordbee is
```

Comment:
Time to finish that description :smile:

Reply:
I mean, to be fair, in the existential sense, is it _really_ wrong? 🤔 

In all seriousness - big whoops. Will be fixed!